### PR TITLE
Logo should be svg, not text

### PR DESCRIPTION
**Pull request by Excellent-coding**
https://linear.app/find-labs/issue/ENG-29/logo-should-be-svg-not-text

### CODE REVIEW

File: apps/web-app/src/components/Navbar.tsx [hunk 1]
```diff
@@ -1,45 +1,60 @@
...
+          <div className="mr-4 cursor-pointer dark:text-gray-100">
+            Learn more
+          </div>
         </Link>
         <Link href={"https://findlabs.org/docs"}>
           <div className="mr-4 cursor-pointer dark:text-gray-100">Help</div>
         </Link>
       </div>
       <Link href={"/"}>
         <div className="flex cursor-pointer">
```
Comment (khalil-omer) [inline apps/web-app/src/components/Navbar.tsx:34]: This div is not horizontally centered properly. It is not aligned with the rest of the page https://share.cleanshot.com/4H4CugXWDml95sHih3vp

File: apps/web-app/src/components/Navbar.tsx [hunk 2]
```diff
@@ -1,45 +1,60 @@
...
-          <Image src="/logo.svg" width={32} height={32} alt="logo" />
-          <div className="text-bold text-xl my-auto ml-3 dark:text-gray-100">find</div>
+          <Image
+            src={theme === "dark" ? "/find-logo-white.svg" : "/find-logo.svg"}
```
Comment (khalil-omer) [inline apps/web-app/src/components/Navbar.tsx:36]: This Image tag looks correct, however as you can see it is not working: https://share.cleanshot.com/Mjfmaah8gYsx88wyApAR

**Review by khalil-omer — CHANGES_REQUESTED** [review]

File: apps/web-app/src/components/Navbar.tsx (hunk 1 above)
Comment (Excellent-coding) [reply apps/web-app/src/components/Navbar.tsx:34]: got it

File: apps/web-app/src/components/Navbar.tsx (hunk 2 above)
Comment (Excellent-coding) [reply apps/web-app/src/components/Navbar.tsx:36]: got it

Comment (khalil-omer) [reply apps/web-app/src/components/Navbar.tsx:36]: @Excellent-coding I am still seeing this issue.

File: apps/web-app/src/components/Navbar.tsx (hunk 1 above)
Comment (khalil-omer) [reply apps/web-app/src/components/Navbar.tsx:34]: @Excellent-coding I am still seeing this issue

**Review by khalil-omer — CHANGES_REQUESTED** [review]
Issues are still happening (see comments)

File: apps/web-app/src/components/Navbar.tsx (hunk 2 above)
Comment (Excellent-coding) [reply apps/web-app/src/components/Navbar.tsx:36]: https://prnt.sc/NlGAJAW-7KTo
Its really wired
This is my screen and everything is working as expected
I think this is related with cache, would you please clear your cache on your browser?

File: apps/web-app/src/components/Navbar.tsx (hunk 1 above)
Comment (Excellent-coding) [reply apps/web-app/src/components/Navbar.tsx:34]: https://prnt.sc/NlGAJAW-7KTo
Its really wired
This is my screen and everything is working as expected
I think this is related with cache, would you please clear your cache on your browser?

**Review by khalil-omer — APPROVED** [review]

**Comment by khalil-omer** [comment]
@Excellent-coding I still see the issues even on a different computer, but I am merging anyway and maybe we can debug once merged.

**Pull request merged**